ui/dashboardApp/layout/main/Sider/Banner.module.dark.less

```diff
@@ -0,0 +1,10 @@
+@import '~antd/es/style/themes/dark.less';
```

Comment:
what kind of styles needs a `.dark` variant?

Reply:
The dark styles for app pages and components